src/pollingManager.ts

```diff
@@ -0,0 +1,33 @@
+import config from 'config';
+import { inject, singleton } from 'tsyringe';
+import { Logger } from '@map-colonies/js-logger';
+import { SERVICES } from './common/constants';
+import { TasksManager } from './tasks/models/tasksManager';
+
+export const POLLING_MANGER_SYMBOL = Symbol('tasksFactory');
+
+@singleton()
+export class PollingManager {
+  private readonly expirationDate: number;
```

Comment:
this is bad name as it is more of ttl (in what units?) then date

Reply:
fixed